### PR TITLE
edit abandon claim to abandon support

### PR DESCRIPTION
**Pull request by nestordominguez**
## PR Checklist

- [x ] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [ x] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?

- [ x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

## Fixes

Issue Number: https://github.com/lbryio/lbry-desktop/issues/3505

## What is the current behavior? 
The tooltip in remove icon from transaction list item show abandon claim in his text
## What is the new behavior?
Now show abandon support

### CODE REVIEW

File: ui/component/transactionListTable/internal/transaction-list-item.jsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ class TransactionListItem extends React.PureComponent<Props> {
     if (type === TXN_TYPES.TIP) {
       return <Button button="secondary" icon={ICONS.UNLOCK} onClick={this.abandonClaim} title={__('Unlock Tip')} />;
     }
-    return <Button button="secondary" icon={ICONS.DELETE} onClick={this.abandonClaim} title={__('Abandon Claim')} />;
+    return <Button button="secondary" icon={ICONS.DELETE} onClick={this.abandonClaim} title={__('Abandon Support')} />;
```
Comment (seanyesmunt) [inline ui/component/transactionListTable/internal/transaction-list-item.jsx]: This isn't correct. We still want to show "Abandon Claim" in some cases.

Comment (nestordominguez) [reply ui/component/transactionListTable/internal/transaction-list-item.jsx]: Can you explain to me in which case should be used Abandon claim and Abandon support?

Comment (seanyesmunt) [reply ui/component/transactionListTable/internal/transaction-list-item.jsx]: if "type === TXN_TYPES.SUPPORT"

Comment (nestordominguez) [reply ui/component/transactionListTable/internal/transaction-list-item.jsx]: fixed, thanks

**Comment by seanyesmunt** [comment]
Nice!

**Comment by tzarebczan** [comment]
Thanks for the PR @nestordominguez ! Can we show you some [appreciation](https://lbry.com/faq/appreciation)?

**Comment by nestordominguez** [comment]
Hi thanks, I finished thta issue in less tahn 10 minutes, this is my wallet:  bT67CBafF9i2w9SYHqDyWjrHimtsupa4Wm

Regards!
________________________________
De: Thomas Zarebczan <notifications@github.com>
Enviado: martes, 28 de enero de 2020 23:41
Para: lbryio/lbry-desktop <lbry-desktop@noreply.github.com>
Cc: nestordominguez <oscardom80@hotmail.com>; Mention <mention@noreply.github.com>
Asunto: Re: [lbryio/lbry-desktop] edit abandon claim to abandon support (#3514)


Thanks for the PR @nestordominguez<https://github.com/nestordominguez> ! Can we show you some appreciation<https://lbry.com/faq/appreciation>?

**Pull request merged**